src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -783,7 +735,8 @@ class BinaryCompactObject : public DomainCreator<3> {
   std::array<double, 2> initial_size_map_accelerations_{
       std::numeric_limits<double>::signaling_NaN(),
       std::numeric_limits<double>::signaling_NaN()};
-  std::array<std::string, 2> size_map_function_of_time_names_;
+  std::array<std::string, 2> size_map_function_of_time_names_{
```

Comment:
I think these added members can be `const`, maybe even `static`? Not sure which is better. @wthrowe?

Reply:
Probably `static`, but I'll wait for @wthrowe's opinion first before I change those.